hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/scale/ITestS3AInputStreamPerformance.java

```diff
@@ -242,6 +242,7 @@ protected void logTimePerIOP(String operation,
 
   @Test
   public void testTimeToOpenAndReadWholeFileBlocks() throws Throwable {
+    skipIfClientSideEncryption();
```

Comment:
why are we turning this off ? Performance? seek() perf?

Reply:
Uses Landsat data that isn't encrypted which causes failures.

Reply:
ok